target/linux/armsr/armv7/config-6.6

```diff
@@ -1,7 +1,6 @@
 CONFIG_ALIGNMENT_TRAP=y
 CONFIG_ARCH_32BIT_OFF_T=y
 CONFIG_ARCH_MIGHT_HAVE_PC_PARPORT=y
-CONFIG_ARCH_MMAP_RND_BITS=8
```

Comment:
This is needed because the target base configuration sets it to 18, see:
```
target/linux/armsr/armv7/config-6.6:CONFIG_ARCH_MMAP_RND_BITS=8
target/linux/armsr/armv8/config-6.6:CONFIG_ARCH_MMAP_RND_BITS=18
target/linux/armsr/config-6.6:CONFIG_ARCH_MMAP_RND_BITS=18
```

Reply:
sounds broken. config-6.6 has CONFIG_64BIT but armv7 is a 32-bit platform.

Reply:
armsr/armv7/config-6.6 overwrites the settings from armsr/config-6.6 again